platform/source/LocalFileSystem.cpp

```diff
@@ -14,6 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#if defined(__ARMCC_VERSION)
+#include <arm_compat.h>
```

Comment:
In commit "semihosting: add compat header where is required", should this commit include the addition of this header to mbed_retarget.cpp as well? Shall we squash "semihosting: add compat header where is required" with "retarget: move compat header for ARMClang prior any cmsis headers"? The reason for both commits are the same, right?

Reply:
I don't see these commits were squashed yet.